src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1331,28 +1331,38 @@ private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(INewDocumentActi
                     var data = builder.CreateReader();
                     builder.Reset();
 
-                    if (data.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata) &&
-                        metadata.TryGet(DocumentItem.ExportDocumentType.Key, out string type))
+                    if (data.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata))
                     {
-                        if (type != DocumentItem.ExportDocumentType.Attachment)
+                        if (metadata.TryGet(Constants.Documents.Metadata.Collection, out string collectionName))
                         {
-                            var msg = $"Ignoring an item of type `{type}`. " + data;
-                            if (_log.IsOperationsEnabled)
-                                _log.Operations(msg);
-                            _result.AddWarning(msg);
-                            continue;
+                            if (collectionsToOperate != null && collectionsToOperate.Contains(collectionName) == false)
```

Comment:
null and count = 0, should have the same result? how it works during export?
you need to convert here (beginning of the method?) collectionsToOperate to a Hashset<string>(StringComparer.OridinalIgnoreCase)

Reply:
BW, what happens to timeseries, attachments, counters, etc when we are exporting/importing collection X ?